.github/sync-repo-settings.yaml

```diff
@@ -0,0 +1,22 @@
+rebaseMergeAllowed: true
+squashMergeAllowed: true
+mergeCommitAllowed: false
+branchProtectionRules:
+- pattern: master
+  isAdminEnforced: false
+  requiredStatusCheckContexts:
+    - 'Kokoro CI - Java 8'
```

Comment:
@lesv should Java 11 be in this list?

Reply:
It should be, but probably isn't.  @averikitsch

Reply:
It is there - so yep - Java 11 should pass